internal/api-tests/builder_test.go

```diff
@@ -67,11 +67,27 @@ func updateScenario[T client.Object](title string, b Builder[T], cb func(b Build
 }
 
 func canCreate[T client.Object](title string, b Builder[T]) {
```

Comment:
🚨 👎  if you have introduced `canCreateSkr` ... functions then rename these to `canCreateKcp` ...

Reply:
That will introduce changes to this pull request that are out of scope.
I will do it in first thing after this is merged.

Reply:
actually, changed it in this PR